SwaprHQ/stackly-ui
# [STK-75][FEAT] - Add Toast component

## Fixes: [STK-75](https://linear.app/swaprdev/issue/STK-75/add-toast-component)

### Description

* Updated Modal logic to opt-out the background opacity
* Created the Toast component
* Updated UI page where we display Stackly components

### Preview
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/162c3db4-3f15-4725-8162-72ec45796340)
https://feat-stk-75-add-toast-compon.stackly-ui.pages.dev/ui



### How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to the Stackly UI components page (`https://feat-stk-75-add-toast-compon.stackly-ui.pages.dev//ui`) and check for the Modals section, you will find a "Toast" button.
